mllib/src/main/scala/org/apache/spark/mllib/recommendation/MatrixFactorizationModel.scala

```diff
@@ -17,14 +17,20 @@
 
 package org.apache.spark.mllib.recommendation
 
-import java.lang.{Integer => JavaInteger}
-
-import org.jblas.DoubleMatrix
+import java.lang.{ Integer => JavaInteger }
 
 import org.apache.spark.SparkContext._
-import org.apache.spark.api.java.{JavaPairRDD, JavaRDD}
+import org.apache.spark.api.java.{ JavaPairRDD, JavaRDD }
 import org.apache.spark.rdd.RDD
 
+import org.apache.spark.util.collection.Utils
+import org.apache.spark.util.BoundedPriorityQueue
+
+import scala.Ordering
```

Comment:
By organizing imports you mean same package imports will move to one right ?

Old: 

import org.apache.spark.mllib.linalg.Vectors
import org.apache.spark.mllib.linalg.Vector
import org.apache.spark.mllib.linalg.BLAS

New:

import org.apache.spark.mllib.linalg.{Vectors, Vector, BLAS}